service/history/events/cache.go

```diff
@@ -93,16 +93,18 @@ func NewGlobalCache(
 	historyManager persistence.HistoryManager,
 	logger log.Logger,
 	metricsClient metrics.Client,
+	maxSizeInKb uint,
```

Comment:
Is it possible to make the value type for size consistent? I think I've seen three types: `uint`, `uint32` and `uint64`.